dip-0027.md

```diff
@@ -0,0 +1,163 @@
+| - | - | - | - |
+| version | uint8_t | 1 | Asset Unlock version number. Currently set to 1 |
+| index | uint64 | 8 | The index of the transaction |
+| fee | uint32 | 4 | The miner's fee in duffs |
+| signHeight | uint32 | 4 | The height of the Core chain known by Platform at the moment of the Asset Unlock signing (Core ChainLock height) |
+| quorumHash | uint256 | 32 | Quorum hash. Set by the current validator. Must refer to an active quorum at `signHeight` |
+| quorumSig | uint768 | 96 | Threshold signature recovered during Platform Consensus. Must be a signature for the full transaction, not just the payload. |
+
+#### Signing Asset Unlock Transactions
+
+An active Platform validator quorum quorum must sign Asset Unlock transactions per the process described in [DIP-7](./dip-0007.md). The signing session parameters are:
+
+| DIP-7 parameter | Value |
+|-|-|
+| Request ID | `SHA256(SHA256("plwdtx", index))` |
+| Message hash | The asset lock transaction, with the `quorumSig` field set to zeros, hashed as described in [DIP-2](./dip-0002.md#serialization-hashing-and-signing) |
+
+For reference, the following table shows some example index values and their request IDs. See the [provided script](dip-0027/dip-0027-request-id-calc.py) for example code:
+
+| **Index** | **Request ID** |
+|-----------|----------------|
+| 101       | fcc76a643c5c668244fdcef09833955d6f4b803fa6c459f7732983c2332389fd |
+| 123456789 | ebd9d75ad72184bec8e8f25a499eb5386ba564eb7ce70a4a3b9db652599d6d72 |
+
+### Client verification
+
+All clients must verify the quorumSig field before accepting the transaction. If the quorum is recent and the transaction is not yet mined, light clients should attempt to verify the quorumSig by retrieving the quorum public key as described in DIP-4 and verifying the signature. If the quorum is not recent and the transaction is not mined, the transaction should be ignored until it is mined to prevent attacks on the light client.
+
+### Withdrawal Completion
+
+Since Asset Unlock transactions do not have inputs, they are not eligible for InstantSend. Once mined into a block and subsequently ChainLocked, they are final and identical to all other outputs. A withdrawal should be considered complete when the corresponding Asset Unlock transaction is finalized on the Core chain.
+
+Asset Unlock transactions might not be mined for multiple reasons. For example, the quorum who signed a transaction expired, Core fees were too low, or the withdrawal limit was reached. To handle this situation, Asset Unlock transactions have an expiration period based on the quorum lifetime. Transactions are considered invalid if they are not signed by either the active quorum or the most recent inactive quorum. In this case, the issuer can retry the withdrawal. To ensure that expired transactions are not included in the Core chain, Asset Lock transactions are refused once the block height exceeds _signHeight_ by 48 or more (i.e., height > _signHeight_ + 48).
```

Comment:
Change to match v22
```suggestion
Asset Unlock transactions might not be mined for multiple reasons. For example, the quorum who signed a transaction expired, Core fees were too low, or the withdrawal limit was reached. To handle this situation, Asset Unlock transactions have an expiration period based on the quorum lifetime. Transactions are considered invalid if they are not signed by either the active quorum set or the previous active set. In this case, the issuer can retry the withdrawal. To ensure that expired transactions are not included in the Core chain, Asset Lock transactions are refused once the block height exceeds _signHeight_ by 48 or more (i.e., height > _signHeight_ + 48).
```

Reply:
@knst so you're saying this would be more accurate?

> the active quorum set or the most recently replaced (removed/deactivated?) quorum

Reply:
> so you're saying this would be more accurate?

> the active quorum set or the most recently replaced (removed/deactivated?) quorum

yes, that sounds right. @PastaPastaPasta ?

Reply:
@PastaPastaPasta re-open if you disagree